### PR TITLE
Add IncorrectDelimiter check name to the message

### LINKED ISSUE #121: IncorrectDelimiter: Show check name in message
Right now the message looks like this:
```
.env:2 The FOO-BAR key has incorrect delimiter
```

Need to add the check name to the message:
```
.env:2 IncorrectDelimiter: The FOO-BAR key has incorrect delimiter
```

### PR DESCRIPTION
**Pull request by undef1nd**
Closes #121 

#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/mgrachev/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [X] Tests have been updated;

### CODE REVIEW

File: src/checks/incorrect_delimiter.rs [hunk 1]
```diff
@@ -19,7 +21,7 @@ impl Check for IncorrectDelimiterChecker {
         if key.trim().chars().any(|c| !c.is_alphanumeric() && c != '_') {
             return Some(Warning::new(
                 line.clone(),
-                self.template.replace("{}", &key),
+                format!("{}: {}", self.name, self.template.replace("{}", &key)),
```
Comment (mgrachev) [inline src/checks/incorrect_delimiter.rs:24]: Could you please move this line to the `message` function? For example:
https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/unordered_keys.rs#L11

**Comment by mgrachev** [comment]
@undef1nd 👋Thank you for your contribution! 👍 

I have left a comment. Could you please fix it?

File: src/checks/incorrect_delimiter.rs (hunk 1 above)
Comment (undef1nd) [reply src/checks/incorrect_delimiter.rs]: Yeah, I've added it.

**Comment by mgrachev** [comment]
@undef1nd Thank you! 🚀

**Pull request merged**